molecule/redhat/molecule.yml

```diff
@@ -17,17 +16,32 @@ lint: |
   flake8
 platforms:
   # Amazon Linux does not appear to offer Xfce
-  # - name: amazonlinux2
-  #   image: amazonlinux:2
-  - name: fedora32
-    image: fedora:32
-  - name: fedora33
-    image: fedora:33
+  # - name: amazonlinux2_systemd
+  #   image: geerlingguy/docker-amazonlinux2-ansible:latest
+  #   privileged: yes
+  #   volumes:
+  #     - /sys/fs/cgroup:/sys/fs/cgroup:ro
+  #   command: /lib/systemd/systemd
+  #   pre_build_image: yes
+  - name: fedora32_systemd
+    image: geerlingguy/docker-fedora32-ansible:latest
+    privileged: yes
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    command: /lib/systemd/systemd
+    pre_build_image: yes
+  - name: fedora33_systemd
+    image: geerlingguy/docker-fedora33-ansible:latest
+    privileged: yes
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    command: /lib/systemd/systemd
+    pre_build_image: yes
 provisioner:
   name: ansible
   inventory:
     host_vars:
-      debian9:
+      debian9_systemd:
```

Comment:
Since this is pared down to _just_ RedHat related distributions should this `debian9_systemd`-fu be here?

Reply:
I have erred on the side of leaving it in until it is removed en masse when we drop support for Debian 9, but I can remove it here if you think it's important.

Reply:
See commit d248aa7.